conans/test/functional/toolchains/apple/test_xcodedeps_components.py

```diff
@@ -0,0 +1,191 @@
+import os
+import platform
+import textwrap
+
+import pytest
+
+from conans.test.utils.tools import TestClient
+
+
+@pytest.mark.skipif(platform.system() != "Darwin", reason="Only for MacOS")
+@pytest.mark.tool_cmake
+@pytest.mark.tool_xcodebuild
+@pytest.mark.tool_xcodegen
+def test_xcodedeps_components():
```

Comment:
I really miss a little explanation about the folder structure and dependency graph created by this test. It's not necessary something huge, but, at least, what we expect running this test.

Reply:
Yes, you are right, this test deserver more explanation 👍